CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 [Unreleased]: https://github.com/rust-marker/marker/compare/v0.4.0...HEAD
+[0.4.2]: https://github.com/rust-marker/marker/releases/tag/v0.4.2
+[0.4.1]: https://github.com/rust-marker/marker/releases/tag/v0.4.1
```

Comment:
My bad, I wish someday this becomes autogenerated

Btw. the first line with `Unreleased` link should be updated to `0.4.2` as well